packages/terra-icon/package.json

```diff
@@ -20,11 +20,11 @@
   },
   "homepage": "https://github.com/cerner/terra-core#readme",
   "devDependencies": {
-    "csvtojson": "^1.1.4",
-    "jsdom": "^9.11.0",
+    "csvtojson": "^2.0.8",
+    "jsdom": "^15.1.1",
     "one-cerner-style-icons": "https://github.com/cerner/one-cerner-style-icons/archive/1.21.1.tar.gz",
```

Comment:
Should we update this too? There are new icons 1.23 is out.
https://github.com/cerner/one-cerner-style-icons/releases

Reply:
Yeah, lets go ahead and pull that in. cc/ @amichaelparker

Reply:
Updated: [838e8ed](https://github.com/cerner/terra-core/pull/2451/commits/838e8ede78621db79bb557ac1d7fbc11ae47b2f2). It added some new icons and changed others, so tests and example pages were also updated.